helm-charts/support/values.yaml

```diff
@@ -24,6 +29,13 @@ prometheus:
   pushgateway:
     enabled: false
   server:
+    ingress:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-type: basic
+        nginx.ingress.kubernetes.io/auth-secret: prometheus-ingress-auth
```

Comment:
TODO: This secret needs to be created and consume the values passed.

Reply:
@consideRatio oh yeah, I just forgot to git add that file! done :)